packages/dashboard-frontend/src/pages/WorkspaceDetails/index.tsx

```diff
@@ -223,9 +209,6 @@ export class WorkspaceDetails extends React.PureComponent<Props, State> {
     const workspaceName = workspace.name;
     const { inlineAlertConversionError, showInlineAlertRestartWarning } = this.state;
 
```

Comment:
Variables are assigned values but never used.

```suggestion
```

Reply:
I have fixed it.